mongoose/UserSchema.ts

```diff
@@ -28,11 +31,11 @@ const UserSchema = new mongoose.Schema<User>({
     lastName: String,
     email: {type: String, required: true},
     type: {type: String, enum: ['AVERAGE', 'CRITIC', 'BUSINESS'], required: true},
-    businessId: String,
+    business: {type: Schema.Types.ObjectId, ref: "RestaurantModel"},
     profilePhoto: String,
     headerImage: String,
-    accountType: {type: String, default: 'PERSONAL', enum: ['PERSONAL', 'ACADEMIC', 'PROFESSIONAL']},
-    maritalStatus: {type: String, default: 'SINGLE', enum: ['MARRIED', 'SINGLE', 'WIDOWED']},
+    accountType: {type: String, default: AccountType.Personal, enum: ['PERSONAL', 'ACADEMIC', 'PROFESSIONAL']},
```

Comment:
I am not sure whether we should keep the accountType. Maybe we can set it as optional or take it away. The accountType field is redundant now since we defined our own type already.

Reply:
I agree- I think we should remove accountType if we will be using the userType to identify the account/user types

Reply:
Sounds good - I'll remove it